pennylane/interfaces/__init__.py

```diff
@@ -12,6 +12,423 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-This subpackage defines functions convert quantum tapes to interface with different machine
-learning libraries.
+This subpackage defines functions for interfacing devices' batch execution
+capabilities with different machine learning libraries.

```

Comment:
Note: this file was moved without modification from `pennylane/interfaces/batch/__init__.py`.